requirements-dev.txt

```
@@ -17,4 +17,4 @@ pytest-cov>=2.11.1  # via -r requirements-dev.in
 pytest>=6.2.2       # via -r requirements-dev.in
 ipython>=8.10.0     # via -r requirements-dev.in, pinned by Snyk to avoid SNYK-PYTHON-IPYTHON-3318382
 pydocstyle>=6.1.0
-setuptools>=65.5.1  # not directly required, pinned by Snyk to avoid SNYK-PYTHON-SETUPTOOLS-3113904
+setuptools>=70.0.0  # not directly required, pinned by Snyk to avoid SNYK-PYTHON-SETUPTOOLS-3113904
```

We noticed some issues with setuptools==71.X that caused our Azure resources to fail to connect to the driver, and found the error resulted from a Snyk upgrade without an upper bound. 
```suggestion
setuptools~=70.3.0  # not directly required, pinned by Snyk to avoid SNYK-PYTHON-SETUPTOOLS-3113904
```

---

[PR here](https://github.com/CrowdStrike/falconpy/pull/1203) to fix